schemas/stsci.edu/asdf/0.1.0/time/time.yaml

```diff
@@ -0,0 +1,232 @@
+      # have nested lists.
+
+    - $ref: ../core/ndarray
+
+    - type: object
+      properties:
+        value:
+          description: |
+            The value(s) of the time.
+
+          anyOf:
+            - type: string
+            - type: number
+            - type: array
+            - $ref: ../core/ndarray
+
+        format:
+          description: |
+            The format of the time.
+
+            If not provided, the the format should be guessed from the
+            string from among the following unambiguous options:
+            `iso`, `byear`, `jyear` and `yday`.
+
+            The supported formats are:
+
+            - `iso`: ISO 8601 compliant date-time format
+              `YYYY-MM-DDTHH:MM:SS.sss...`.  For example,
+              `2000-01-01 00:00:00.000` is midnight on January 1,
+              2000.  The `T` separating the date from the time
+              section is optional.
+
+            - `yday`: Year, day-of-year and time as
+              `YYYY:DOY:HH:MM:SS.sss...`. The day-of-year (DOY) goes
+              from 001 to 365 (366 in leap years). For example,
+              `2000:001:00:00:00.000` is midnight on January 1,
+              2000.
+
+            - `byear`: Besselian Epoch year, eg. `B1950.0`.  The `B`
+              is optional if the `byear` format is explicitly
+              specified.
+
+            - `jyear`: Julian Epoch year, eg. `J2000.0`.  The `J` is
+              optional if the `jyear` format is explicitly
+              specified.
+
+            - `decimalyear`: Time as a decimal year, with integer
+              values corresponding to midnight of the first day of
+              each year. For example 2000.5 corresponds to the ISO
+              time `2000-07-02 00:00:00`.
+
+            - `jd`: Julian Date time format. This represents the
+              number of days since the beginning of the Julian
+              Period. For example, 2451544.5 in `jd` is midnight on
+              January 1, 2000.
+
+            - `mjd`: Modified Julian Date time format. This
+              represents the number of days since midnight on
+              November 17, 1858. For example, 51544.0 in MJD is
+              midnight on January 1, 2000.
+
+            - `gps`: GPS time: seconds from 1980-01-06 00:00:00 UTC
+              For example, 630720013.0 is midnight on January 1,
+              2000.
+
+            - `unix`: Unix time: seconds from 1970-01-01 00:00:00
+              UTC. For example, 946684800.0 in Unix time is midnight
+              on January 1, 2000.  [TODO: Astropy's definition of
+              UNIX time doesn't match POSIX's here.  What should we
+              do for the purposes of ASDF?]
+
+            - `cxcsec`: Chandra X-ray Center seconds from 1998-01-01
+              00:00:00 TT. For example, 63072064.184 is midnight on
+              January 1, 2000.  [TODO: Is this general enough to
+              include?]
+
```

Comment:
I thought about including it, but I think YAML format provides a cleaner way to do this (by using a dictionary) and there's probably not much point to require ASDF libraries to implement a specialized parser for the FITS time scale convention.  As for the pyasdf implementation of this, it can save a FITS format time as a dictionary with the value (in iso) and scale.  I sort of have "reduce the amount of specialized parsing required" as a guiding principle in ASDF,  but if there's a good argument for including FITS scaled times verbatim, I could be convinced otherwise.